test/Tests/src/Data/Range_Spec.enso

```diff
@@ -10,7 +10,7 @@ import Standard.Base.Errors.Illegal_State.Illegal_State
 import Standard.Base.Runtime.Ref.Ref
 
 from Standard.Test import Test, Test_Suite
-import Standard.Test.Extensions
+from Standard.Test.Extensions import all
```

Comment:
Since we now do `from ... Extensions ... import/export all` in Standard.Base/Main.enso, it should be safe to revert similar lines

Reply:
Reverted in ca9bcc81c22136c572e252d878810694a30cfdc0